### PR TITLE
[BigQueryWorkloadTester] Adding config files and the classes used to parse them into Java

### PR DESCRIPTION
**Pull request by ldanielmadariaga**
[BigQueryWorkloadTester] Adding config files and the classes used to parse them into Java

### CODE REVIEW

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  public Boolean isRatioBasedBenchmark() {
+    return isRatioBasedBenchmark;
+  }
+
+  public void setIsRatioBasedBenchmark(Boolean isRatioBasedBenchmark) {
+    this.isRatioBasedBenchmark = isRatioBasedBenchmark;
+  }
+
+  public List<WorkloadSettings> getWorkloads() {
+    return workloads;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: this.workloads for clarify?

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  public String getOutputFileFolder() {
+    return outputFileFolder;
+  }
+
+  public void setOutputFileFolder(String outputFileFolder) {
+    this.outputFileFolder = outputFileFolder;
+  }
+
+  public Boolean isRatioBasedBenchmark() {
+    return isRatioBasedBenchmark;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: this.isRatioBasedBenchmark for clarity

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  public Integer getConcurrencyLevel() {
+    return concurrencyLevel;
+  }
+
+  public void setConcurrencyLevel(Integer concurrencyLevel) {
+    this.concurrencyLevel = concurrencyLevel;
+  }
+
+  public String getOutputFileFolder() {
+    return outputFileFolder;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: "return this." for clarity

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    // Eagerly read queries to ensure the loading does not affect the benchmark reads.
+    instance.workloads.forEach(w -> w.getQueries());
+
+    return true;
+  }
+
+  public Integer getConcurrencyLevel() {
+    return concurrencyLevel;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: "return this." for clarity

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+  private String name;
+  private String projectId;
+  private String cloudCredentialsFile;
+  private String outputFileName;
+  private List<String> queries;
+  private List<String> queryFiles;
+
+  public String getName() {
+    return name;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java]: "return this." for clarity

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public String getProjectId() {
+    return projectId;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java]: "return this." for clarity

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+    return optional;
+  }
+
+  public void setCloudCredentialsFile(String cloudCredentialsFile) {
+    this.cloudCredentialsFile = cloudCredentialsFile;
+  }
+
+  public String getOutputFileName() {
+    return outputFileName;
```
Comment (EricBeach) [inline BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java]: "return this." for clarity

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java (hunk 1 above)
Comment (ldanielmadariaga) [reply BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: Done

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java (hunk 2 above)
Comment (ldanielmadariaga) [reply BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: Done

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java (hunk 3 above)
Comment (ldanielmadariaga) [reply BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: Done

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java (hunk 4 above)
Comment (ldanielmadariaga) [reply BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/Configuration.java]: Done

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java (hunk 6 above)
Comment (ldanielmadariaga) [reply BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java]: Done

File: BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java (hunk 7 above)
Comment (ldanielmadariaga) [reply BigQueryWorkloadTester/src/main/java/com/google/cloud/pontem/config/WorkloadSettings.java]: Done

**Pull request merged**